app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="wrapper">
+  <h2>Add New Recipe</h2>
+  <%= form_with model: @recipe, url: user_recipes_path(user_id: current_user.id), method: :post do |f| %>
+    <div class="field">
+      <%= f.label :name, class: "form-label" %><br />
+        <%= f.text_field :name, autofocus: true, autocomplete: "name" , class: "form-control" %>
+    </div>
+    <div class="field">
+      <%= f.label :preparation_time, class: "form-label" %><br />
+        <%= f.text_field :preparation_time, autofocus: true, autocomplete: "preparation_time" ,
+          class: "mb-3 form-control" %>
+    </div>
+    <div class="field">
+      <%= f.label :cooking_time, class: "form-label" %><br />
+        <%= f.text_field :cooking_time, autofocus: true, autocomplete: "cooking_time" , class: "mb-3 form-control" %>
+    </div>
+    <div class="field">
+      <%= f.label :description, class: "form-label" %><br />
+        <%= f.text_area :description, autofocus: true, autocomplete: "description" , class: "mb-3 form-control" %>
+    </div>
+    <div class="mb-3 form-check">
+      <%= f.radio_button :public, true %>
+        <%= f.label 'Public' , :public=> 'True' %>
+
+          <%= f.radio_button :public, false %>
+            <%= f.label 'Private' , :public=> 'False' %>
```

Comment:
- [ ] When creating a private recipe, I can still view it on the public list and yet I am not supposed to.  For example, I made Krabby Patty public and Secret formula private as seen in the example bellow but I can see them both on the public recipes tab. Please fix that.
![image](https://github.com/cosywasswa/Foods-App/assets/45460192/858ab1bb-e78f-4a97-b178-68c7308ab48c)